app/routes/qr+/$qrId_.contact-owner.tsx

```diff
@@ -23,6 +24,7 @@ export const NewReportSchema = z.object({
 
 export const action = async ({ request, params }: ActionArgs) => {
   assertIsPost(request);
+  const { organizationId } = await requireAuthSession(request);
```

Comment:
This won't work correctly in this case. This action is performed when you scan an asset that does NOT belong to you. So if we use your `organizationId` it won't work. Thats why in this case we find it based on the `ownerId`. 
I think in this case just querying the asset by its own ID is sufficient, because it doesn't matter to which organization it depends, we just need the specific asset that was scanned.

Reply:
thanks for pointing, i have fixed it.